### DISCUSSION TITLE
Change the onboarding branch to your own, instead of main

### DISCUSSION THREAD

**Discussion by Alexsidin** [renovatebot/renovate#32202] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Maybe I don't understand something, so I decided to ask here, I want when creating an onboarding PR it to be merged into my branch (for example, feature/TASK-11111), and not into main.
I have already changed RENOVATE_BASE_BRANCHES, but I haven't figured out how to merge into the onboarding branch yet

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Not possible - renovate must have a config in the default branch, so that's what the PR targets. Just push a config manually to your feature branch if you need.
